specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2022-11-01-preview/examples/RedisEnterpriseSkusList.json

```diff
@@ -0,0 +1,319 @@
+                  "value": true
+                },
+                {
+                  "name": "FDsAvailable",
+                  "value": true
+                }
+              ]
+            },
+            "skuDetails": {
+              "name": "Enterprise_DRAM_E2"
+            }
+          },
+          {
+            "resourceType": "Microsoft.Cache/redisEnterprise",
+            "locationInfo": {
+              "location": "westus2",
+              "capabilities": [
+                {
+                  "name": "ZoneAware",
+                  "value": true
+                },
+                {
+                  "name": "FDsAvailable",
+                  "value": true
+                }
+              ]
+            },
+            "skuDetails": {
+              "name": "Enterprise_DRAM_E3"
+            }
+          },
+          {
+            "resourceType": "Microsoft.Cache/redisEnterprise",
+            "locationInfo": {
+              "location": "westus2",
+              "capabilities": [
+                {
+                  "name": "ZoneAware",
+                  "value": true
+                },
+                {
+                  "name": "FDsAvailable",
+                  "value": true
+                }
+              ]
+            },
+            "skuDetails": {
+              "name": "Enterprise_DRAM_E4"
+            }
+          },
+          {
+            "resourceType": "Microsoft.Cache/redisEnterprise",
+            "locationInfo": {
+              "location": "westus2",
+              "capabilities": [
+                {
+                  "name": "ZoneAware",
+                  "value": true
+                },
+                {
+                  "name": "FDsAvailable",
+                  "value": true
+                }
+              ]
+            },
+            "skuDetails": {
+              "name": "Enterprise_DRAM_E5"
+            }
+          },
+          {
+            "resourceType": "Microsoft.Cache/redisEnterprise",
+            "locationInfo": {
+              "location": "westus2",
+              "capabilities": [
+                {
+                  "name": "ZoneAware",
+                  "value": true
+                },
+                {
+                  "name": "FDsAvailable",
+                  "value": true
+                }
+              ]
+            },
+            "skuDetails": {
+              "name": "Enterprise_SSD_S1"
+            }
+          },
+          {
+            "resourceType": "Microsoft.Cache/redisEnterprise",
+            "locationInfo": {
+              "location": "westus2",
+              "capabilities": [
+                {
+                  "name": "ZoneAware",
+                  "value": true
+                },
+                {
+                  "name": "FDsAvailable",
+                  "value": true
+                }
+              ]
+            },
+            "skuDetails": {
+              "name": "Enterprise_SSD_S2"
```

Comment:
>Enterprise_SSD_S2 [](http://example.com/codeflow?start=23&length=17)

Only valid SKU names for us to return in our example are:

EnterpriseFlash_F1500 
EnterpriseFlash_F300 
EnterpriseFlash_F700 
Enterprise_E10 
Enterprise_E100 
Enterprise_E20 
Enterprise_E50 

same as Microsoft.Cache/redisEnterprise type [in latest REST API]